src/animations.ts

```diff
@@ -0,0 +1,35 @@
+import { css } from 'styled-components';
+
+export const animations = css`
```

Comment:
You can also use `keyframes`, I believe (same below): https://styled-components.com/docs/basics#animations
```suggestion
export const animations = keyframes`
```